fal/models/team.py

```diff
@@ -9,5 +9,7 @@ class Team(Base):
 
     id = Column(Integer, primary_key=True)
     season_id = Column(String, ForeignKey('season.id'))
-    season = relationship("Season", back_populates='teams')
     name = Column(String)
+
+    season = relationship("Season", back_populates='teams')
```

Comment:
This is pretty minor, but shouldn't 'teams' be "teams" to stick with the double quote usage?

Reply:
i think double/single quotes are interchangable in python? but will change this

Reply:
They are interchangable, but i thought it was better style to stick to one in a file on codebase.